### DISCUSSION TITLE
Scan number

### DISCUSSION THREAD

**Discussion by ABRai11** [vdemichev/DiaNN#1092] [Q&A] [answered]
Hi,

How to get scan number for a particular precursor of interest, so we can inspect the raw file for fragments.

**Comment by vdemichev** [comment, marked as answer]
Hi,

I would recommend to match the scan with the retention time that corresponds to the apex RT value (in minutes) that DIA-NN reports in the main report.

You can also check a particular precursor using the visualisation capabilities of DIA-NN https://github.com/vdemichev/DiaNN?tab=readme-ov-file#visualisation

Best Regards,
Vadim

**Comment by ABRai11** [comment]
Hi,

Thanks for your reply.
